include/evmc/evmc.h

```diff
@@ -411,7 +411,10 @@ typedef bool (*evmc_account_exists_fn)(struct evmc_context* context, const evmc_
 /**
  * Get storage callback function.
  *
- * This callback function is used by a VM to query the given contract storage entry.
+ * This callback function is used by a VM to query the given account storage entry.
+ * The VM MUST make sure that the account exists. This requirement is only a formality because
```

Comment:
Is it really needed for get_storage? `@return` spec below states that null bytes are returned in case account does not exist (so it's not undefined behavior)

Reply:
Yes, you're right.

Reply:
Changed.